processor/processor.go

```diff
@@ -1345,7 +1345,7 @@ func getDiffMetrics(inPU, pu string, inCountMetadataMap map[string]MetricMetadat
 	return diffMetrics
 }
 
-func (proc *Handle) processJobsForDest(partition string, subJobs subJob, parsedEventList [][]types.SingularEventT) *transformationMessage {
```

Comment:
removing the third parameter. Didn't look like it added any value by keeping it.